tests/wiTest.cpp

```diff
@@ -989,6 +989,9 @@ TEST_CASE("WindowsInternalTests::UniqueHandle", "[resource][unique_any]")
 
         // address
         REQUIRE(*spMoveHandle.addressof() == handleValue);
+        REQUIRE(*spMoveHandle.put() == nullptr);
+        *spMoveHandle.put() = ::CreateEventEx(nullptr, nullptr, CREATE_EVENT_INITIAL_SET, 0);

```

Comment:
consider calling the W version directly, avoiding the macros

Reply:
The whole test suite uses the macros, the same line but for `operator&` (which I copied from) also uses the macro.